### PR TITLE
Fix small font on sts page - temporary

### LINKED ISSUE #2254: [BUG] small normal word on LowerECAM Status page
**Mod Version**
 latest master

**Describe the bug**

#1964 fixed only SELF TEST, need to fix on STS page

**To Reproduce**

1. spawn on aircraft
2. select lower ecam to STS page
3.

**Expected behavior**

**Actual behavior**

very small word

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by Kimbyeoungjang**
Fixes #2254

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by Kimbyeoungjang** [comment]
sry for that dirty commit. 
just see last 2af97ba commit

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -203,4 +203,4 @@
 1. [ECAM] Added cockpit door video - @Benjozork (Benjamin Dupont)
 1. [MISC] Standby Instrument stays ON if emergency power should be available, bug fixes - @2hwk (2Cas#1022 on discord)
 1. [CDU] Full +/- button functionality - @lhoenig (Lukas Hoenig)
-1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
\ No newline at end of file
+1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
```
Comment (beheh) [inline .github/CHANGELOG.md:206]: You're changing the last line here. Please make sure your IDE does not mess with files unrelated to your commit.

**Review by beheh — CHANGES_REQUESTED** [review]

File: .github/CHANGELOG.md (hunk 1 above)
Comment (Kimbyeoungjang) [reply .github/CHANGELOG.md:206]: Idk why github(or my ide) always make a new line on last :(. Anyway, i remove that line

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -203,4 +204,4 @@
...
```
Comment (Kimbyeoungjang) [inline .github/CHANGELOG.md:207]: can you guys fix it? why github web can't remove that last enter line??
also ide cmd is not working also god damnn

**Comment by ChruutvoLuzi** [comment]
Category: QA Tester
Name: ChruutvoLuzi | SD-PB.ch#8902
Date of testing: 8.12.2020
Version of the sim: 1.11.6.0
PR Tested: #2255 
Object of test: Fix Small Normal on Status Page

Testing technique: Try out
Steps you've made to test the Pull Request:
1. Spawned C&D at Finkenwerder EDHI
2. Started up the AC
3. Checked the Status page

Flight notes (If required): none

Issues (If Any): none
Overall Rating: Good
Conclusions of the testing: Works as described, the Font has a normal size now
Microsoft Flight Simulator Screenshot 2020 12 08 - 14 45 10 25

**Pull request merged**